### PR TITLE
pypy3.9 7.3.16

### PR DESCRIPTION
**Pull request by cho-m**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by p-linnane** [comment]
Opened an upstream issue re: the checksums: https://github.com/pypy/pypy/issues/4919

**Comment by p-linnane** [comment]
Upstream confirmed that the files were modified a few hours after release.

**Comment by p-linnane** [comment]
Tests need to be updated and we need a long build as well.

**Review by chenrui333 — APPROVED** [review]

File: Formula/p/pypy3.9.rb [hunk 1]
```diff
@@ -135,8 +140,16 @@ def post_install
...
     site_packages(libexec).parent.install_symlink site_packages(HOMEBREW_PREFIX)
 
-    # Tell distutils-based installers where to put scripts
+    # Create a scripts folder in the prefix and symlink it as libexec/bin.
+    # This is needed as setuptools' distutils ignores our distutils.cfg.
+    # Also create symlinks for original binaries we moved in case they are needed.
     scripts_folder.mkpath
+    libexec.install_symlink scripts_folder => "bin" unless (libexec/"bin").exist?
+    (libexec/"pypybin").each_child do |f|
+      scripts_folder.install_symlink f unless (scripts_folder/f.basename).exist?
```
Comment (cho-m) [inline Formula/p/pypy3.9.rb:149]: I might need to check what happens on revision bump / upgrade as may be symlinked via Cellar path.

**Comment by chenrui333** [comment]
there is a new release now 7.3.16

**Review by iMichka — APPROVED** [review]

**Pull request merged**